.travis.yml

```diff
@@ -1,45 +0,0 @@
-sudo: yes
-dist: xenial
-
-language: python
-
-cache: pip
-
-matrix:
-  include:
-    - python: "3.5"
-    - python: "3.6"
-    - python: "3.7"
-    - python: "3.8"
-    - name: Requirements
-      env: TOXENV=requirements
-      python: "3.5"
-      addons: null
-      before_script: null
-    - name: Coding Style
-      env: TOXENV=style
-      python: "3.5"
-      addons: null
-      before_script: null
-    - name: Sanitizer Configuration
-      env: TOXENV=sanitizer
```

Comment:
These three "Requirements", "Coding Style", and "Sanitizer Configuration" should be preserved in the GitHub Actions.

Something like this should work for them:
https://github.com/andersinno/django-analog/blob/eb5f481cbf399c359fb52174a184ace0bab71da7/.github/workflows/test.yml#L15-L17